app/app.go

```diff
@@ -604,64 +654,14 @@ func NewNibiruApp(
 	app.SetBeginBlocker(app.BeginBlocker)
 	app.SetEndBlocker(app.EndBlocker)
 
-	app.UpgradeKeeper.SetUpgradeHandler(
```

Comment:
Remove upgrade handler due to no ICS27 supporting
Ref: https://docs.cosmos.network/master/core/upgrade.html#running-migrations